nms/packages/magmalte/fbc_js_core/ui/host/Organizations.js

```diff
@@ -300,65 +331,63 @@ function Organizations(props: Props) {
             }}
           />
         </Grid>
-        <Route
-          path={relativePath('/new')}
-          render={() => (
-            <OrganizationDialog
-              addUser={addUser}
-              setAddUser={() => setAddUser(true)}
-              onClose={() => {
-                setAddUser(false);
-                history.push(relativeUrl(''));
-              }}
-              onCreateOrg={org => {
-                let newOrg = null;
-                axios
-                  .post('/host/organization/async', org)
-                  .then(() => {
-                    enqueueSnackbar('Organization added successfully', {
-                      variant: 'success',
-                    });
-                    axios
-                      .get(`/host/organization/async/${org.name}`)
-                      .then(resp => {
-                        newOrg = resp.data.organization;
-                        if (newOrg) {
-                          setOrganizations([...organizations, newOrg]);
-                          setAddingUserFor(newOrg);
-                        }
-                      });
-                  })
-                  .catch(error => {
-                    setAddUser(false);
-                    history.push(relativeUrl(''));
-                    enqueueSnackbar(error?.response?.data?.error || error, {
-                      variant: 'error',
-                    });
-                  });
-              }}
-              onCreateUser={user => {
+        <OrganizationDialog
+          edit={false}
+          hideAdvancedFields={props.hideAdvancedFields}
+          organization={null}
+          user={null}
+          open={showOrganizationDialog}
+          addUser={addUser}
+          setAddUser={() => setAddUser(true)}
+          onClose={() => {
+            setShowOrganizationDialog(false);
+            setAddUser(false);
+          }}
+          onCreateOrg={org => {
+            let newOrg = null;
+            axios
+              .post('/host/organization/async', org)
+              .then(() => {
+                enqueueSnackbar('Organization added successfully', {
+                  variant: 'success',
+                });
                 axios
-                  .post(
-                    `/host/organization/async/${
-                      addingUserFor?.name || ''
-                    }/add_user`,
-                    user,
-                  )
-                  .then(() => {
-                    enqueueSnackbar('User added successfully', {
-                      variant: 'success',
-                    });
-                    setAddingUserFor(null);
-                    history.push(relativeUrl(''));
-                  })
-                  .catch(error => {
-                    enqueueSnackbar(error?.response?.data?.error || error, {
-                      variant: 'error',
-                    });
+                  .get(`/host/organization/async/${org?.name ?? ''}`)
+                  .then(resp => {
```

Comment:
No  because I reset the tabs on close https://github.com/magma/magma/pull/12394/files#diff-b927e1ba7650dd78506eb4417cbb6821d2ff49e38e5f34e2a22dab7119999202R346

Reply:
It did happen when I played with this locally. The docs say:
onClose - Callback fired when the component requests to be closed.
so it might not be called when it's closed from the outside.

Reply:
I was just setting the open prop to false so not firing the onClose callback